### PR TITLE
Fix incoming recents

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview for *hopeful-hamilton-4090b9* ready!

Built with commit 95a4009689ad3f4313ba56d21e4c966fec0fca29

https://deploy-preview-669--hopeful-hamilton-4090b9.netlify.com

File: test/journeys/specs/recents/dataApi/basic.js [hunk 1]
```diff
@@ -186,5 +209,17 @@ describe('Widget Recents', () => {
         'does not show call button');
       });
     });
+
+    describe('incoming call', () => {
+      it('should display incoming call screen', () => {
+        browserRemote.element(meetElements.meetWidget).element(meetElements.callButton).waitForVisible();
+        browserRemote.element(meetElements.callButton).click();
+        browserLocal.waitUntil(() =>
+          browserLocal.element(`${elements.answerButton}`).isVisible(),
```
Comment (moriahmaney) [inline test/journeys/specs/recents/dataApi/basic.js]: Could this just be ` browserLocal.element(elements.answerButton).isVisible()`?

File: test/journeys/specs/recents/global/basic.js [hunk 2]
```diff
@@ -224,14 +248,25 @@ describe('Widget Recents', () => {
...
       'does not show call button');
     });
   });
 
+  describe('incoming call', () => {
+    it('should display incoming call screen', () => {
+      browserRemote.element(meetElements.meetWidget).element(meetElements.callButton).waitForVisible();
+      browserRemote.element(meetElements.callButton).click();
+      browserLocal.waitUntil(() =>
+        browserLocal.element(`${elements.answerButton}`).isVisible(),
```
Comment (moriahmaney) [inline test/journeys/specs/recents/global/basic.js]: Same thing here

File: wdio.conf.js [hunk 3]
```diff
@@ -90,6 +90,12 @@ exports.config = {
     recents: [
       './test/journeys/specs/recents/**/*.js'
     ],
+    recentsGlobal: [
+      './test/journeys/specs/recents/global/**/*.js'
+    ],
+    recentsData: [
+      './test/journeys/specs/recents/dataApi/**/*.js'
+    ],
```
Comment (moriahmaney) [inline wdio.conf.js]: What's the reasoning for these extra suites? Just wondering as we don't usually do that

Comment (adamweeks) [reply wdio.conf.js]: laziness for local testing

File: test/journeys/specs/recents/dataApi/basic.js (hunk 1 above)
Comment (adamweeks) [reply test/journeys/specs/recents/dataApi/basic.js]: copypasta, let me see where that was from and fix both

**Pull request merged**